app/controllers/admin_request_controller.rb

```diff
@@ -40,8 +40,10 @@ def show
                             :name_to => @info_request.user_name,
                             :name_from => AlaveteliConfiguration::contact_name,
                             :info_request_url => request_url(@info_request, :only_path => false)}
-    @request_hidden_user_explanation = render_to_string(:template => "admin_request/hidden_user_explanation",
-                                                        :locals => vars_for_explanation)
+    @request_hidden_user_explanation =
+      render_to_string(template: "admin_request/hidden_user_explanation",
```

Comment:
If we make the template a text template rather than html we probably won't need this change.